apps/academy/src/components/Header.tsx

```diff
@@ -36,42 +41,47 @@ const PageHeader: FunctionComponent = () => {
   const { pathname } = router;
 
   return (
-    <header className="app-container absolute left-0 right-0 top-0 z-50 flex items-start justify-between px-8 pt-8 md:flex-row">
-      <div className="hidden lg:flex ">
+    <header
+      className="absolute top-2 z-40 flex w-full items-center justify-between bg-transparent  px-[15px]
+     pt-[30px] md:px-[30px]  md:pt-[20px]"
+    >
+      <div className="z-50  hidden lg:flex">
         {pathname === "/" || pathname === "/tracks" || pathname === "/fundamentals" ? (
-          <div className="nav-division flex flex-col gap-y-6">
-            <div className="flex justify-between">
-              <Link href="/">
-                <Image src="/academy_logo.svg" width={150} height={40} alt="Academy Logo" />
-              </Link>
-              {/* <ThemeToggleButton hidden={isConnected} /> */}
-            </div>
-            <div className="mx-auto">
-              <TopBar menus={topbarNavMenus} />
-            </div>
+          <div className="flex justify-between">
+            <Link href="/">
+              <Image src="/academy_logo.svg" width={150} height={40} alt="Academy Logo" />
+            </Link>
+            {/* <ThemeToggleButton hidden={isConnected} /> */}
           </div>
         ) : (
           <div className="flex items-center justify-around gap-36 text-white lg:flex lg:justify-between lg:gap-5 lg:self-stretch">
-            <div className="lg:ml-8 lg:flex lg:basis-[0%] lg:flex-col lg:items-stretch">
+            <div className="lg:items-stretch9 lg:ml-8 lg:flex lg:basis-[0%] lg:flex-col">
```

Comment:
@rohann06 I believe there is a typo here: `lg:items-stretch9` the `lg:items-stretch` class has an extra `9` at the end, can we fix it?